frontend/test/metabase-smoketest/admin_setup.cy.spec.js

```diff
@@ -0,0 +1,643 @@
+import {
+  restore,
+  signInAsAdmin,
+  USERS,
+  signOut,
+  signInAsNormalUser,
+} from "__support__/cypress";
+import { createMasonryCellPositioner } from "react-virtualized";
+
+const new_user = {
+  first_name: "Barb",
+  last_name: "Tabley",
+  username: "new@metabase.com",
+};
+
+describe("smoketest > admin_setup", () => {
+  before(restore);
+
+  describe("successful setup by admin", () => {
+    beforeEach(signInAsAdmin);
+
+    it("should add a new database", () => {
+      // *** I don't have any faux databases to hook up to. Are there some in the system already?
+      // *** Should eventually include BigQuery, Druid, Google Analytics, H2, MongoDB, MySQL/Maria DB, PostgreSQL, Presto, Amazon Redshift, Snowflake, Spark SQL, SQLite, SQL Server
+
+      cy.visit("/");
+
+      // Navigate to page
+
+      cy.get(".Icon-gear")
+        .first()
+        .click();
+      cy.findByText("Admin").click();
+
+      cy.contains("Metabase Admin");
+      cy.contains("dashboard").should("not.exist");
+
+      cy.findByText("Databases").click();
+
+      cy.contains("Sample Dataset");
+      cy.contains("Updates").should("not.exist");
+
+      cy.findByText("Add database").click();
+
+      cy.contains(
+        "Automatically run queries when doing simple filtering and summarizing",
+      );
+
+      // Adds new database
+
+      // cy.findByText("H2").click();
+      // cy.findByText("PostgreSQL").click();
+      // cy.findByLabelText("Name").type("Postgre Database");
+      // cy.findByLabelText("Host").type("");
+      // cy.findByLabelText("Port").type("");
+      // cy.findByLabelText("Database name").type("");
+      // cy.findByLabelText("Database username").type("");
+      // cy.findByLabelText("Database password").type("");
+      // // *** check that toggles are correct
+      // cy.findByLabelText("Additional JDBC connection string options").type("");
+      // cy.findByText("Save").click();
+    });
+
+    it("should setup email", () => {
+      // *** maybe using something like maildev)
+
+      cy.findByText("Settings").click();
+      cy.findByText("Email").click();
+
+      cy.contains("Email address you want to use as the sender of Metabase.");
+      cy.contains("Sample Database").should("not.exist");
+
+      // Email info
+      // cy.findByLabelText("SMTP HOST").type("smtp.gmail.com");
+      // cy.findByLabelText("SMTP PORT").type("465");
+      // cy.findByText("SSL").click();
+      // cy.findByLabelText("SMTP USERNAME").type(""); // *** enter email here
+      // cy.findByLabelText("SMTP PASSWORD").type(""); // *** enter password here
+      // cy.findByLabelText("FROM ADDRESS").type("metabase@metabase.com");
+      // cy.findByText("Save changes").click();
```

Comment:
Let's go ahead and enable this. We can use fake info for now and just verify that we can click save and that the "Send test email" button pops up.

You'll probably need to use `cy.findByPlaceholderText` instead of `findByLabelText` since the labels aren't proper `<input>` labels.

```javascript
      cy.findByPlaceholderText("smtp.yourservice.com")
        .type("localhost")
        .blur();
```